### PR TITLE
Fix crash with pipeline describe when using inline taskSpec

### LINKED ISSUE #1302: pipeline describe crash when pipeline uses inline taskSpec
# Versions and Operating System

- Kubernetes version:

  **Output of `kubectl version`:**

  ```
  Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-13T13:28:09Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
  Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.0+7070803", GitCommit:"70708036fc265771f8d0a45598209018a8b9bd3e", GitTreeState:"clean", BuildDate:"2020-12-05T12:01:07Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
  ```

- Tekton Pipeline version:

  **Output of `tkn version` or `kubectl get pods -n tekton-pipelines -l app=tekton-pipelines-controller -o=jsonpath='{.items[0].metadata.labels.version}'`**

  ```
  Client version: 0.16.0
  Pipeline version: v0.16.3
  Triggers version: v0.8.1
  ```

  Second command failed btw, but I installed Red Hat OpenShift Pipelines Operator 1.2.3

  ```
  error: error executing jsonpath "{.items[0].metadata.labels.version}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
  	template was:
		{.items[0].metadata.labels.version}
  	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}
  ```

- Operating System:

  ```
  Fedora Linux 33
  tkn downloaded from GitHub releases (tkn_0.16.0_Linux_x86_64.tar.gz)
  ```

# Expected Behavior

Command shows the list of tasks

# Actual Behavior

Command crashs

# Steps to Reproduce the Problem

1. Apply this pipeline:

```yaml
apiVersion: tekton.dev/v1beta1
kind: Pipeline
metadata:
  name: inline-task-spec
spec:
  tasks:
    - name: show-version
      taskSpec:
        steps:
          - name: show-version
            image: node
            script: |
              pwd
              node --version
```

2. Run `tkn pipeline describe inline-task-spec`

Current output incl. crash (status code 1):

```
Name:        inline-task-spec
Namespace:   christoph

📦 Resources

 No resources

⚓ Params

 No params

📝 Results

 No results

📂 Workspaces

 No workspaces

🗒  Tasks

Error: template: Describe Pipeline:61:37: executing "Describe Pipeline" at <$t.TaskRef.Name>: nil pointer evaluating *v1beta1.TaskRef.Name
```

# Additional Info
None

### PR DESCRIPTION
**Pull request by chmouel**
# Changes

When the task is using a taskSpec we return task name as EMBEDDED

Closes #1302

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [X] Run the code checkers with `make check`
- [X] Regenerate the manpages, docs and go formatting with `make generated`
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```release-note
Fix crash with embedded taskrun in pipeline when running `tkn pipeline describe`
```

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
/meow

File: pkg/formatted/task.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+package formatted
+
+import (
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+func GetTaskRefName(task *v1beta1.PipelineTask) string {
+	if task.TaskRef != nil {
+		return task.TaskRef.Name
+	}
```
Comment (vinamra28) [inline pkg/formatted/task.go:24]: can we add a test for this?

Comment (chmouel) [reply pkg/formatted/task.go:24]: it's explicit with https://github.com/tektoncd/cli/pull/1303/files#diff-37b860e7d4510beb113f11d5a6aa216ead6b58c455f495cb45c49ea71042d6bcR549 isnt it ?

i can add a implicit test but i don't think that add value or improve coverage,

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

File: pkg/formatted/task.go (hunk 1 above)
Comment (vinamra28) [reply pkg/formatted/task.go:24]: yeah I agree but I see that each entry in the formatted package has a corresponding test hence said in order to maintain the consistency

Comment (chmouel) [reply pkg/formatted/task.go:24]: Oh yeah fair enough, let maybe add that then..

**Pull request merged**